cmd/mysterium_server/command_run/command.go

```diff
@@ -38,6 +38,18 @@ func (cmd *CommandRun) Run(options CommandOptions) (err error) {
 		return err
 	}
 
+	// if no identity can be selected, lets create a new one
+	if providerId == nil {
```

Comment:
This logic is very tight with selection process, and could easily fit in `identityHandler` aswel